libs/model/src/chainEventSignatures.ts

```diff
@@ -0,0 +1,15 @@
+export const deployedNamespaceEventSignature =
+  '0x8870ba2202802ce285ce6bead5ac915b6dc2d35c8a9d6f96fa56de9de12829d5';
+
+export const communityStakeTradeEventSignature =
+  '0xfc13c9a8a9a619ac78b803aecb26abdd009182411d51a986090f82519d88a89e';
+
+export const proposalEventSignatures = [
+  '0x11a0b38e70585e4b09b794bd1d9f9b1a51a802eb8ee2101eeee178d0349e73fe',
+  '0x712ae1383f79ac853f8d882153778e0260ef8f03b504e2866e0593e04d2b291f',
```

Comment:
Wouldn't it be easier to read if this was a record w/ human readable event names?

Reply:
Indeed we could convert to using enums here. I'll review your mapping PR first and then create a ticket once I've seen your implementation and how we can move forward.